tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,24 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## October 18, 2022
+
+**General changes**
+
+* For Dedicated Tier clusters, the minimum storage size of a TiKV or TiFlash node is changed from 500 GiB to 200 GiB. This will be more cost-effective for users whose workloads are in small data volumes.
+
+    For more details, see [TiKV node storage](/tidb-cloud/size-your-cluster.md#tikv-node-storage) and [TiFlash node storage](/tidb-cloud/size-your-cluster.md#tiflash-node-storage).
+
+* Introduce online contracts to customize TiDB Cloud subscriptions and meet compliance requirements.
+
+    A [**Contract** tab](/tidb-cloud/tidb-cloud-billing.md#contract) is added to the Billing page of the TiDB Cloud console. If you have agreed with our sales on a contract and received an email to process the contract online, you can go to the **Contract** tab to review and accept the contract. To learn more about contracts, feel free to [contact our sales](https://www.pingcap.com/contact-us/).
```

Comment:
The `contract` link anchor will be valid after https://github.com/pingcap/docs/pull/10818 is merged.